src/core/caniuse.js

```diff
@@ -0,0 +1,212 @@
+/*
+Module: "core/caniuse"
+Adds a caniuse support table for a "feature" #1238
+
+`conf.caniuse = { feature: key }` =>
+  .. the table is added before Copyright
+Optional settings:
+  `conf.caniuse.browsers` list of caniuse supported browser names
+    to be shown in the table or "ALL"
+    default: ["chrome", "firefox", "safari", "edge"]
+  `conf.caniuse.versions` number of browser versions to show
+  `conf.caniuse.maxAge` (in ms) local response cache duration
+  `conf.caniuse.apiURL` URL from where to fetch stats.
+    use {FEATURE} as placeholder in URL to replace it by a feature name
+*/
+
+import { semverCompare } from "core/utils";
+import IDBCache from "core/idb-cache";
+import { pub } from "core/pubsubhub";
+import "deps/hyperhtml";
+import caniuseCss from "deps/text!core/css/caniuse.css";
+
+export const name = "core/caniuse";
+
+export async function run(conf) {
```

Comment:
So, can I use is not critical spec content, so maybe we could do something like:

```JS
export async function run(conf, doc, cb) {
  cb(); // this is low priority, so let's keep going
  if (!conf.caniuse) {
    return; // nothing to do.
  }
  document.head.appendChild(hyperHTML`<style>${caniuseCss}</style>`);
  const cache = new IDBCache("respec-caniuse", ["caniuse"]);
  normalizeConf(conf);
  const { caniuse } = conf;

  if (!caniuse.feature) {
    return; // no feature to show
  }
  // Insert placeholder dt/dd into head
  // kick off fetch here too...
  await Promise.all([doc.respecIsReady, cache.ready(), /* fetch */]);

  await canIUse(
    caniuse.feature,
    document.querySelector(".head dl").lastChild,
    cache,
    caniuse
  );
}
```

Reply:
What's the purpose of doing a `fetch` here?
We should do a fetch only if the cache is invalid so as to avoid requests on github